minecode/visitors/generic.py

```diff
@@ -137,16 +137,26 @@ def map_directory_listed_package(package_url):
     "pkg:generic/samba@.*",
     "pkg:generic/syslinux@.*",
     "pkg:generic/toybox@.*",
-    "pkg:generic/uclibc@@.*",
+    "pkg:generic/uclibc@.*",
     "pkg:generic/uclibc-ng@.*",
     "pkg:generic/util-linux@.*",
     "pkg:generic/wpa_supplicant@.*",
     "pkg:generic/ipkg@.*",
+    "pkg:generic/linux@.*",
+    "pkg:generic/mtd-utils@.*",
+    "pkg:generic/barebox@.*",
+    "pkg:generic/e2fsprogs@.*",
+    "pkg:generic/udhcp@.*",
+    "pkg:generic/miniupnpc@.*",
+    "pkg:generic/miniupnpd@.*",
+    "pkg:generic/minissdpd@.*",
+    "pkg:generic/erofs-utils@.*",
 ]
 
-
-@priority_router.route(*DIR_SUPPORTED_PURLS)
-def process_request_dir_listed(purl_str):
+# Indexing some generic PURLs requires a GitHub API token.
```

Comment:
I think it would be good to write about the `GH_TOKEN` environment variable in the installation section of the readme.

Reply:
Done